bentoml/_internal/external_typing/tensorflow.py

```diff
@@ -0,0 +1,356 @@
+# type: ignore[reportMissingTypeStubs]
+import os
```

Comment:
@aarnphm For TensorFlow which is not typed officially, we may just implement the pyi file for it.